test/factories/mapping_rule_factory.rb

```diff
@@ -2,8 +2,10 @@
 
 FactoryBot.define do
   factory :mapping_rule do
-    sequence(:id, 1)
-    sequence(:integration_id, 1)
+    association :integration, factory: :integration
+    association :input, factory: :product
+    association :output, factory: :product # TODO: FactoryBot is upset about this
```

Comment:
@alexmgrant I opted to skip the factory validation tests for now, but FactoryBot was unhappy with this.

I figure since y'all are reworking things skipping was fine, but let's try and make sure whatever comes next gets our factory validation test back on